lighthouse-core/config/default-config.js

```diff
@@ -140,6 +140,7 @@ const defaultConfig = {
     'without-javascript',
     'metrics/first-contentful-paint',
     'metrics/first-meaningful-paint',
+    'metrics/first-contentful-paint-3g',
```

Comment:
yeah good call

Reply:
we only need it in mobile right @paulirish ?

Reply:
Was this change only applied to mobile? I was testing with this version and all the desktop scores went down dramatically for first contentful paint. Are we sure that this metrics is not accidentally included in the desktop fcp score calculation?
@brendankenny @paulirish @patrickhulce

Reply:
@dgargi desktop scoring changes are because of https://github.com/GoogleChrome/lighthouse/commit/161519a60dea58afc5f42f450925b966da05ffcb

Reply:
Hi @patrickhulce thank you so much for the reply. Am I correct to think that the Google Pagespeed API `https://developers.google.com/speed/pagespeed/insights` calls lighthouse to compute the score?

I am asking because I downloaded the latest lighthouse release 4.1.0 (2019-01-28) and compared it to release 4.0.0-beta (2018-11-29) and the scores returned are the same for 
`node lighthouse-cli <url> --disable-device-emulation --throttling-method=provided --view `

This is making me think that nothing is wrong with lighthouse nor with the website being tested but that something changed with `https://www.googleapis.com/pagespeedonline/v5/runPagespeed`

Does this make sense?

Reply:
if you're comparing to PSI you need to use the same settings PSI uses.

If you have the repo/code locally, `node ./lighthouse-cli <url> --config-path=./lighthouse-core/config/lr-mobile-config.js`. Replace `lr-mobile-config.js` with `lr-desktop-config.js` for the desktop quivalent.

Reply:
Oh thank you. This is invaluable information.
I tried to run the command you have given me 
`node ./lighthouse-cli <url> --config-path=./lighthouse-core/config/lr-desktop-config.js` with the older version and compared the result to the new one.
I noticed something odd. The latest version is saying in the results
`Emulated Desktop, Simulated Slow 4G network`
whereas tag v3.2.1 says
`Emulated Nexus 5X, Simulated Slow 4G network`

Does that mean that in version v3.2.1 the configuration `lr-desktop-config.js` was not applied?

Reply:
@patrickhulce seems this issue has put the internet in a frenzy. There are several threads in the [PSI group](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/pagespeed-insights-discuss). 
Any ideas?

Reply:
thanks @dgargi  there's a bug that is being investigated that also affected mobile scores (#7151) this is probably what most folks are seeing

Reply:
@dgargi @patrickhulce thanks so much for the quick investigation. I filed #7151 and if you need more information, let me know. Thanks!